src/window-service.js

```diff
@@ -268,14 +268,16 @@
                     resizable: false,
                     minHeight: 500,
                     minWidth: 230,
-                    maximizable: false
+                    maximizable: false,
+                    shadow: true
                 });
             } else {
                 _window.updateOptions({
                     resizable: true,
                     minHeight: 510,
                     minWidth: 918,
-                    maximizable: true
+                    maximizable: true,
+                    shadow: true
                 });
             }
         }
```

Comment:
I don't think you need to update the shadow options here, as they were never set to false.